File: Python/Mahasiswa.py
# Import file.
from Sivitas_Akademik import Sivitas_Akademik


# membuat kelas Mahasiswa dengan pewarisan dari kelas Sivitas_Akademik
class Mahasiswa (Sivitas_Akademik):
    # Membuat Private attributes.
    __nim = 0
    __prodi = ""
    __fakultas = ""

    # Constructor with default initialization
    def __init__(self, nama="", nik="", Jenis_Kelamin="", asal_univ="", email="", nim=0, prodi="", fakultas=""):
        # memanggil konstruktor dari kelas induk (superclass)
        super().__init__(nik, nama, Jenis_Kelamin, asal_univ, email)
        self.__nama = nama
        self.__nim = nim
        self.__prodi = prodi
        self.__fakultas = fakultas

    # Setter and Getter methods for private attributes
    # Set Nim
    def setNim(self, nimMahasiswa):
        self.__nim = nimMahasiswa

    # Get Nim
    def getNim(self):
        return self.__nim

    # Set Prodi
    def setProdi(self, programStudi):
        self.__prodi = programStudi

    # Get Prodi
    def getProdi(self):
        return self.__prodi

    # Set Fakultas
    def setFakultas(self, fakultas):
        self.__fakultas = fakultas

    # Get Fakultas
    def getFakultas(self):
        return self.__fakultas

File: PHP/Mahasiswa.php
<?php
require('Sivitas_Akademik.php');

// membuat kelas Mahasiswa dengan pewarisan dari kelas Sivitas_Akademik
class Mahasiswa extends Sivitas_Akademik
{
    // Membuat atribut Private dari kelas Sivitas_Akademik
    private $nim = '';
    private $prodi = '';
    private $fakultas = '';

    # Constructor with default initialization
    public function __construct($nama = '', $nik = '', $gender = '', $asal_univ = '',  $nim = '', $prodi = '', $fakultas = '',  $email_edu = '',  $image,)
    {
        # memanggil konstruktor dari kelas induk(superclass)
        parent::__construct($nama, $nik, $gender, $image, $asal_univ, $email_edu);
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
    }

    # Setter and Getter methods for private attributes
    # Set Nim
    public function setNim($nimMahasiswa)
    {
        $this->nim = $nimMahasiswa;
    }

    # Get Nim
    public function getNim()
    {
        return $this->nim;
    }

    # Set Prodi
    public function setProdi($programStudi)
    {
        $this->prodi = $programStudi;
    }

    # Get Prodi
    public function getProdi()
    {
        return $this->prodi;
    }

    # Set Fakultas
    public function setFakultas($fakultas)
    {
        $this->fakultas = $fakultas;
    }

    # Get Fakultas
    public function getFakultas()
    {
        return $this->fakultas;
    }
}
